topi/include/topi/nn.h

```diff
@@ -83,6 +83,31 @@ inline tvm::Tensor leaky_relu(const tvm::Tensor& t,
     tag);
 }
 
+/*!
+* \brief Creates an operation that performs a parametric rectified linear unit
+*
+* \param x The input data tensor
+* \param w The input weight tensor
+* \param name The name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor whose op member is the relu operation
+*/
+template <typename T>
+inline tvm::Tensor prelu(const tvm::Tensor& x,
+                              const tvm::Tensor& w,
+                              std::string name = "tensor",
+                              std::string tag = kBroadcast) {
+  auto m = broadcast_mul(x, w);
```

Comment:
Note: channel-wise multiplication cannot simply be broadcast_mul for nchw if we allow the tensor w need to be 1D